### DISCUSSION TITLE
Feature 1.17

### DISCUSSION THREAD

**Discussion by vorinvyd** [PowerNukkit/PowerNukkit#1222] [General]
Added Full update Caves & Clifs Part 1

**Comment by MonoAdrian23** [comment]
I think the Devs are working on It.
But i think its useless because its much more better for waiting 1.18 to add all these items

**Comment by dmartin015** [comment]
I think they should also add an update in the Nether

**Comment by MonoAdrian23** [comment]
This is Size S for me.
A converter or LevelDB Support is much more important.
And a own MobPlugin.
